### DISCUSSION TITLE
Does yubikey challenge / response change everytime db is saved?

### DISCUSSION THREAD

**Discussion by NqNfC** [keepassxreboot/keepassxc#5929] [Q&A] [answered]
The keepassxc FAQ says: "We use the database's master key as challenge ... " and it also says "(we) gain the advantage that the required response changes every time you save the database". 

Here does the "master key" mean the master password of the db? If so, the challenge remains the same most of the time, unless one changes the password. In that case, how would the response changes?

**Comment by phoerious** [comment, marked as answer]
It's the master seed, not the key really. The master seed is a random byte string.
I fixed the description on the website.

**Reply by NqNfC** [reply]
phoerious, Thanks for the quick response, and for updating the FAQ. IMHO it would be even better if FAQ says the random byte string changes every time when people save. Based on your answer and the FAQ context, I am 99.9% sure that's the case, but to spell it out can help new people be assure.

To be able to change the challenge often is definitely assuring to users like me. To always use the same challenge/response doesn't fully utilize the its ability, and is more vulnerable.

**Reply by phoerious** [reply]
The seed changes every time you save the database. No need for any user intervention.
